drivers/clock_control/nrf_power_clock.c

```diff
@@ -228,6 +228,21 @@ static struct clock_control_async_data *list_get(sys_slist_t *list)
 	return async_data;
 }
 
+static inline void anomaly_132_workaround(nrf_clock_task_t start_tsk)
+{
```

Comment:
Could maybe do

```c
#if CONFIG_NRF52_ANOMALY_132_DELAY_US != 0
// Function body
#endif
```

and remove the `CONFIG_NRF52_ANOMALY_132_DELAY_US` in the `if (...)`. Undefined macros evaluate to 0.

Reply:
done

Reply:
As `CONFIG_FOO` macros are sometimes defined to an empty sequence Ibecause that's enough to get `#ifdef ` to work), the most robust way to do this sort of test is:
```
#if (CONFIG_NRF52_ANOMALY_132_DELAY_US - 0)
// feature
#endif
```
This will work for undefined, defined to empty sequence, defined to 0, and defined to non-zero integer, which covers the normal range of possibilities.  Only the last (defined to non-zero integer) causes the feature to be included.

Reply:
done

Reply:
Kconfig itself skips outputting macros for `n`-valued symbols and symbols with unsatisfied dependencies at least.

The Linux kernel just does stuff like `#if CONFIG_PGTABLE_LEVELS >= 3`.